R/visualisation_recipe.R

```diff
@@ -2,9 +2,17 @@
 #'
 #' This function prepares objects for visualisation by returning a list of
 #' layers with data and geoms that can be easily plotted using for instance
-#' `ggplot2`. See the documentation for your object's class:
-#'
+#' `ggplot2`.
+#' \cr\cr
```

Comment:
No need to use ` \cr\cr` any more since we use markdown syntax to write `.Rd` docs.

Reply:
Without that it puts the text in the Details section

Reply:
Maybe because there is no `#' @description` at the top, and so `roxygen2` is using the heuristic of separate sections per paragraph?